tests/e2e-leg-6/verify-server-logrotate/README.txt

```diff
@@ -0,0 +1,12 @@
+This is a test suite that verifies the server logrotate
+Design: https://confluence.verticacorp.com/pages/viewpage.action?spaceKey=DEV&title=Spec+for+logrotate+in+the+Server
```

Comment:
We do not put confluence link on github. Also this not need to be this big we just want a summary of what the test does(see other README.txt